packages/databricks-vscode/src/configuration/BundleConfigReaderWriter.ts

```diff
@@ -68,9 +69,10 @@ export class BundleConfigReaderWriter {
         return target;
     }
 
-    public async getWorkspaceFsPath(target?: BundleTarget) {
-        const filePath = target?.workspace?.file_path;
-        return filePath ? new RemoteUri(filePath) : undefined;
+    public async getWorkspaceFsPath(
```

Comment:
Why async, is `target?.workspace?.file_path` a promise? Should we add `Promise<... | undefined>` to the return type?

Reply:
The idea was to have all the getter functions to have the same interface so that we can easily loop over them and await all.